src/main/java/Du/Deadline.java

```diff
@@ -19,14 +19,14 @@ public Deadline(String name, boolean done, TaskList tasklist, LocalDateTime time
     }
 
     /**
-     * record the Deadline in a certain format to save to the file
+     * records the Deadline in a certain format to save to the file
```

Comment:
One of the JavaDocs requirements is that every function descriptor should start with a capital letter .It is recommended to follow the general guidelines